### PR TITLE
GG-36310 Sql logic tests.

### PR DESCRIPTION
**Pull request by xtern**

### CODE REVIEW

**Review by AMashenkov — APPROVED** [review]

File: modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/h2/TableWithImplicitPkTest.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 GridGain Systems, Inc. and Contributors.
```
Comment (ygerzhedovich) [inline modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/h2/TableWithImplicitPkTest.java:2]: not sure that is required

File: modules/indexing/src/test/java/org/apache/ignite/sqllogic/SqlScriptRunner.java [hunk 2]
```diff
@@ -0,0 +1,805 @@
...
+import org.apache.ignite.internal.util.tostring.GridToStringInclude;
+import org.apache.ignite.internal.util.typedef.F;
+import org.apache.ignite.internal.util.typedef.internal.S;
+import org.apache.ignite.internal.util.typedef.internal.U;
+import org.gridgain.internal.h2.util.StringUtils;
+import org.jetbrains.annotations.NotNull;
+
+import static org.apache.ignite.internal.util.IgniteUtils.byteArray2HexString;
+
+/**
```
Comment (ygerzhedovich) [inline modules/indexing/src/test/java/org/apache/ignite/sqllogic/SqlScriptRunner.java:57]: javadoc

File: modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/h2/TableWithImplicitPkTest.java (hunk 1 above)
Comment (xtern) [reply modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/h2/TableWithImplicitPkTest.java]: Reverted.

File: modules/indexing/src/test/java/org/apache/ignite/sqllogic/ScriptRunnerTestsEnvironment.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ */
+
+package org.apache.ignite.sqllogic;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
```
[the remark this answers is not in the document]
Comment (xtern) [reply modules/indexing/src/test/java/org/apache/ignite/sqllogic/ScriptRunnerTestsEnvironment.java:24]: Thanks, Javadoc added.

File: modules/indexing/src/test/java/org/apache/ignite/sqllogic/ScriptTestRunner.java [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+import org.apache.ignite.spi.discovery.tcp.ipfinder.vm.TcpDiscoveryVmIpFinder;
+import org.apache.ignite.testframework.junits.logger.GridTestLog4jLogger;
+import org.apache.ignite.thread.IgniteThread;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.runner.Description;
+import org.junit.runner.Runner;
+import org.junit.runner.notification.Failure;
+import org.junit.runner.notification.RunNotifier;
+
```
[the remark this answers is not in the document]
Comment (xtern) [reply modules/indexing/src/test/java/org/apache/ignite/sqllogic/ScriptTestRunner.java:48]: Done

**Pull request merged**